### PR TITLE
Add initial admin code for ubuntu packages

### LINKED ISSUE #1099: Ubuntu package is missing new oauth2 config
Ubuntu server(master) package is missing new oauth2 related envs (initial admin token). Maybe we should prompt those (or non-interactive, generate them if possible).

### PR DESCRIPTION
**Pull request by jnummelin**
fixes #1099

### CODE REVIEW

**Comment by jnummelin** [comment]
ping @jakolehm

File: server/packaging/ubuntu/kontena-server/DEBIAN/templates [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Template: kontena-server/initial_admin_code
+Type: string
+Default:
+Description: Initial admin code for Kontena Master
```
Comment (jakolehm) [inline server/packaging/ubuntu/kontena-server/DEBIAN/templates]: Should we say that it can be anything (random string)?

Comment (jnummelin) [inline server/packaging/ubuntu/kontena-server/DEBIAN/templates]: 👍  I'll add that

Comment (jakolehm) [inline server/packaging/ubuntu/kontena-server/DEBIAN/templates]: And we need to update Ubuntu doc page ...

Comment (jnummelin) [inline server/packaging/ubuntu/kontena-server/DEBIAN/templates]: updated.

**Pull request merged**